cas-server-support-saml/src/main/java/org/jasig/cas/support/saml/authentication/principal/GoogleAccountsService.java

```diff
@@ -168,7 +168,7 @@ public boolean isLoggedOutAlready() {
      * @return the SAML response
      */
     private String constructSamlResponse() {
-        final DateTime currentDateTime = DateTime.parse(new ISOStandardDateFormat().getCurrentDateAndTime());
+        final DateTime currentDateTime = new DateTime();
```

Comment:
Fix the build failure please. You now have an unused import for the line you removed.